practices/cicd/CICD.md

```diff
@@ -0,0 +1,53 @@
+# CI/CD
+
+- **Continuous Integration (CI)**: Automated integration of code
+- **Continuous Deployment (CD)**: Automated deployment of software
```

Comment:
@adamschnaare when I think CI/CD I usually think of the spectrum with 3 bench marks.

Continuous Integration - Automated integration of code
Continuous Delivery - Automated integration of code and system; aka we know we _could_ deploy after each commit
Continuous Deployment - Automated deployment of software; we take that next step from Continuous Delivery and actually put something into production on each commit.

The difference between the last two, IMO, requires a high level of maturity with your Logging, Monitoring, Alerting, and Profiling.

Do you think it's worth covering the difference in this?